.github/CHANGELOG.md

```diff
@@ -60,8 +79,6 @@ and change `lightning.gpu` to inherit from `QubitDevice` instead of `LightningQu
 * Implement improved `stopping_condition` method, and make Linux wheel builds more performant.
 [(#77)](https://github.com/PennyLaneAI/pennylane-lightning-gpu/pull/77)
 
```

Comment:
Do we have to remove this? It's kept in previous release changelog.

Reply:
This is the previous changelog.

Reply:
Sorry, I don't know if I got your point.

Reply:
Do you mean the `### Documentation` tags?

Reply:
yes, just make it consistent.